packages/wpcom-checkout/test/introductory-offer.ts

```diff
@@ -0,0 +1,64 @@
+import { ResponseCartProduct } from '@automattic/shopping-cart';
+import { getItemIntroductoryOfferDisplay } from '../src';
+
+const emptyIntroductoryOffer = { introductory_offer_terms: {} } as ResponseCartProduct;
+
+const disabledIntroductoryOffer = {
+	introductory_offer_terms: {
+		enabled: false,
+		interval_count: 3,
+		interval_unit: 'month',
+		reason: '',
+		should_prorate_when_offer_ends: true,
+		transition_after_renewal_count: 0,
+	},
+} as ResponseCartProduct;
+
+const disabledIntroductoryOfferForAReason = {
+	introductory_offer_terms: {
+		enabled: false,
+		interval_count: 3,
+		interval_unit: 'month',
+		reason: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit.',
+		should_prorate_when_offer_ends: true,
+		transition_after_renewal_count: 0,
+	},
+} as ResponseCartProduct;
+
+const enabledIntroductoryOffer = {
+	introductory_offer_terms: {
+		enabled: true,
+		interval_count: 3,
+		interval_unit: 'month',
+		should_prorate_when_offer_ends: true,
+		transition_after_renewal_count: 0,
+	},
+} as ResponseCartProduct;
+
+describe( 'Introductory Offer Utils', () => {
```

Comment:
Since there's nothing else being tested here, I'd just list the function name directly.

```suggestion
describe( 'getItemIntroductoryOfferDisplay()', () => {
```